nc_time_axis/__init__.py

```diff
@@ -89,9 +88,9 @@ def __init__(self, max_n_ticks, calendar, date_unit, min_n_ticks=3):
 
         self.max_n_ticks = max_n_ticks
         self.min_n_ticks = min_n_ticks
-        self._max_n_locator = mticker.MaxNLocator(max_n_ticks, integer=False)
+        self._max_n_locator = mticker.MaxNLocator(max_n_ticks, integer=True)
```

Comment:
As just demonstrated, when plotting the example from the integration test and zooming in multiple times it was able to update the tick labels appropriately but not sub secondly.